analysis/os_reports/01_eol_service_report_charts.R

```diff
@@ -299,6 +299,45 @@ ggsave(deaths_month_plot, dpi = 600, width = 20, height = 10, unit = "cm"
        , filename = "deaths_month_plot.png"
        , path = here::here("output", "os_reports", "eol_service"))
 
+# Use of medications for symptom management  -------------------------------------------
+
+# Medication use by place of death - including all deaths
+eol_med_pod <- df %>%
+  group_by(study_month, pod_ons_new) %>%
+  summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+  bind_rows(df %>%
+              group_by(study_month) %>%
+              summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+              mutate(pod_ons_new = "All"))
+
```

Comment:
Could you add a rounding step like for the other monthly dataframes created. It was added into the main branch after your branch was taken off.